salt/modules/pkgng.py

```diff
@@ -317,7 +317,7 @@ def latest_version(*names, **kwargs):
             name,
             sorted(
                 __salt__['cmd.run'](cmd, python_shell=False, output_loglevel='trace').splitlines(),
-                reverse=True
+                reverse=False
```

Comment:
@abednarik but this is not what your PR description says. While it is OK for BSD, it is not OK for others. Therefore I would do it differently:

```python
# Is this BSD-like packaging system?
reverse = not _is_bsd_packaging_system()
...
    sorted(__salt__['cmd.run'](.....
           reverse=reverse)
```

Reply:
Hi @isbm  thanks for your comments. 

 Let me try to understand here. As far as I understand pkgng.py is only loaded for FreeBSD >= 10,  DragonFly or if someone manually set pkgng. I know we could use a better approach, but at the same time is hard to me to test it outside FreeBSD.
 So maybe I can set a `reverse` var as you suggest, but I will leave it True as default, since is the way it is now and just set it to False when is FreeBSD.

Open to your feedback.
Thanks.

Reply:
@abednarik ahhh, right. I've mistakenly mixed it up with other close-name module. Sorry. Then you could go as this, but please double-check if this does not hurt any other versions.

I am removing the blocker, but please double-check this.